test/controllers/AccessibilityStatementControllerSpec.scala

```diff
@@ -95,4 +97,25 @@ class AccessibilityStatementControllerSpec extends ControllerBaseSpec {
       }
     }
   }
+  "the user is insolvent and not continuing to trade" should {
+
+    "return 403 (Forbidden)" in new Test {
+      override val authResult: Future[Enrolments ~ Option[AffinityGroup]] = Future.successful(new ~(
+        otherEnrolment,
+        Some(Individual)
+      ))
+
+      override def setup: Any = {
```

Comment:
You don't need to have this setup here, or override the `authResult` above. The default action in the trait is to mock a successful enrolment (see the first unit test in this spec).